gdcdictionary/schemas/exposure.yaml

```diff
@@ -291,6 +328,146 @@ properties:
         $ref:
           - "_terms_enum.yaml#/tobacco/common"
 
+  occupation_duration_years:
```

Comment:
maybe a dumb question: is this referring to how long in a person's life they are engaged in the occupation? or which calendar years? if the former, I think max of 89 and min of 0?